core/market/decentralized/src/market.rs

```diff
@@ -203,13 +203,13 @@ impl MarketService {
     pub async fn get_agreement(
         &self,
         agreement_id: &AgreementId,
-        _id: &Identity,
+        id: &Identity,
     ) -> Result<Agreement, AgreementError> {
         // TODO: Authorization
```

Comment:
Is id in select an authorization already?

Reply:
Hmmm we could think about it this way ;)
I'm not sure, that's why I left this comment

Reply:
Ok, if you think so, I will remove this comment